weed/admin/view/app/table_format.go

```
@@ -0,0 +1,72 @@
+// bucketCatalogPath is the path a client uses to reach one bucket, which differs
+// per format because the two are different protocols on different ports.
+func bucketCatalogPath(format, bucket string) string {
+	if isLanceFormat(format) {
+		return fmt.Sprintf("/v1/namespace/%s/list", bucket)
+	}
+	return fmt.Sprintf("/v1/%s/namespaces", bucket)
+}
+
+// catalogPortFor picks the port that serves a bucket. Zero means that server is
+// not running, and the page says so rather than printing a dead address.
+func catalogPortFor(format string, icebergPort, lancePort int) int {
```

golangci-lint (2.12.2)

[error] func catalogPortFor is unused

(unused)